packages/packages/commonwell-cert-runner/src/patient-management.ts

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env node
+import {
+  CommonWell,
+  getIdPatient,
+  RequestMetadata,
+  getId
+} from "@metriport/commonwell-sdk";
+
+import { patient, mergePatient, personStrongId } from "./payloads";
+
+// 2. Patient Management
+// https://commonwellalliance.sharepoint.com/sites/ServiceAdopter/SitePages/Patient-Management-(PIX,-REST).aspx
+
+export async function patientManagement(commonWell: CommonWell, queryMeta: RequestMetadata) {
+  console.log(`>>> D1b: Register a new Patient`);
+  const respD1b = await commonWell.registerPatient(queryMeta, patient);
+  console.log(respD1b);
+
+  console.log(`>>> D2a: Update demographics for a local Patient `);
+  patient.details.name[0].family[0] = "Graham";
+  const patientId = getIdPatient(respD1b);
+  const respD2a = await commonWell.updatePatient(queryMeta, patient, patientId)
```

Comment:
General comment: there's several lines like this one that are missing `;`, is prettier not catching this with auto format?

Reply:
My default formatter was typescript in vscode. Just changed it to prettier